docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
+|                                       | default, Kinto doesn't define a secret for you, and won't start unless   |
+|                                       | you generate one.                                                        |
++---------------------------------------+--------------------------------------------------------------------------+
+| multiauth.policies ``["basicauth",    | `MultiAuthenticationPolicy <https://github.com/mozilla-                  |
+| ]``                                   | services/pyramid_multiauth>`_ is a Pyramid authentication policy that    |
+|                                       | proxies to a stack of other IAuthenticationPolicy objects, to provide a  |
+|                                       | combined auth solution from individual pieces. Simply pass it a list of  |
+|                                       | policies that should be tried in order.                                  |
++---------------------------------------+--------------------------------------------------------------------------+
+| multiauth.policy.basicauth.use        | Python *dotted* path to the authentication policy to use for basicauth.  |
+| ``('cliquet.authentication.           | By default, every couple of login / password will be accepted, meaning   |
+| BasicAuthAuthenticationPolicy')``     | that no account creation is required.                                    |
++---------------------------------------+--------------------------------------------------------------------------+
+| multiauth.authorization_policy        | Python *dotted* path the authorization policy to use for basicAuth.      |
+| ``('cliquet.authorization.            | Letting the default value will require the :term:`principal` to have     |
+| AuthorizationPolicy')``               | access to the required fields.                                           |
++---------------------------------------+--------------------------------------------------------------------------+
+
+Since user identification is hashed in storage, a secret key is required
+in configuration:
+
+.. code-block:: ini
+
+    # cliquet.userid_hmac_secret = b4c96a8692291d88fe5a97dd91846eb4
+
+
+Authentication setup
+::::::::::::::::::::
+
+*Kinto* relies on :github:`pyramid multiauth <mozilla-service/pyramid_multiauth>`_
+to initialize authentication.
+
+Therefore, any authentication policy can be specified through configuration.
+
+For example, using the following example, *Basic Auth*, *Persona* and *IP Auth*
+are enabled:
+
+.. code-block:: ini
+
+    multiauth.policies = basicauth pyramid_persona ipauth
+
+    multiauth.policy.ipauth.use = pyramid_ipauth.IPAuthentictionPolicy
+    multiauth.policy.ipauth.ipaddrs = 192.168.0.*
+    multiauth.policy.ipauth.userid = LAN-user
+    multiauth.policy.ipauth.principals = trusted
+
+
+Similarly, any authorization policies and group finder function can be
```

Comment:
Consider:
> Permission handling and authorisation mechanisms are specified directly via configuration. This allows for customised solutions ranging from very simple to highly complex.

> Since this is all handled via configuration